### ISSUE TITLE
Infrared/Depth data broken on Intel NUC

### ISSUE THREAD

**Issue by UnaNancyOwen** [microsoft/Azure-Kinect-Sensor-SDK#1189]
**Describe the bug**

Infrared/Depth data that using NFOV 2x2 Binned mode on Intel NUC ([NUC8i7HVK](https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc8i7hvk.html)) is broken.
The log doesn't show any warnings and errors.

![capture1](https://user-images.githubusercontent.com/816705/80076692-806e1a80-8587-11ea-8ea2-2c8a455abe72.PNG)

It's connected to USB 3.0 port controlled by Intel USB host controller.
It's connected using the cable that included in Azure Kinect DK package.
Also, even if I change connect to other USB 3.0 port, but it is broken yet. 

![capture2](https://user-images.githubusercontent.com/816705/80077911-21110a00-8589-11ea-9657-19140825de39.PNG)

It's not broken with other modes (NFOV Unbinned, WFOV 2x2 Binned, and WFOV Unbinned).

![capture3](https://user-images.githubusercontent.com/816705/80077453-7d275e80-8588-11ea-8bf2-c303b6fbee3b.PNG)

Also, I've also tried it on multiple machine with the same model number. 
In all of them, this problem is reproduced. Therefor, It's not an individual problem of machine.
And, I've also tried it with other Azure Kinect.
In all of them, this problem is reproduced. Therefor, It's not an individual problem of Azure Kinect.

It works fine with NFOV 2x2 Binned mode when connect same Azure Kinect to another PC.
Therefore, my Azure Kinect is fine.

![2020-04-23_21h42_31](https://user-images.githubusercontent.com/816705/80100767-61817f80-85ab-11ea-95a3-82055e164cb9.png)

What do you think?

**To Reproduce**

1. Launch the Azure Kinect Viewer.
2. Push [Open Device] button.
3. Set the Depth mode to "NFOV Binned".
4. Push [Start] button.

**Desktop (please complete the following information):**
The OS, firmware, driver and BIOS are all up to date.

 - OS with Version: Windows 10 1909
 - SDK Version:  v1.4.0
 - Firmware version:  AzureKinectDK_Fw_1.6.108079014.bin (included in SDK v1.4.0)

**Additional context**

**Comment by wes-b** [comment]
Usually when we see this kind of issue, the device has a bad graphics driver on it. Can you look into that? We currently don't have one of these units to test with.

@rabbitdaxi FYI

**Comment by UnaNancyOwen** [comment]
@wes-b Hi, Thanks for advice!
I have found that the Intel Driver Update Utility does not bring the Graphic Driver up to date. 
Therefor, I have manually updated to the latest graphic driver that donwloaded from AMD.
It is not broken data with NFOV 2x2 Binned mode. Thanks,

I'll leave a note of the driver version and URL for those who have the same environment.
NOTE: It's a WHQL version.

* Adrenalin 2020 Edition 20.2.2 Recommended (WHQL) (3/5/2020 Released)
  https://www.amd.com/en/support/graphics/radeon-rx-vega-series/radeon-rx-vega-series/radeon-rx-vega-64
